test/junit/scala/tools/nsc/interpreter/TabulatorTest.scala

```diff
@@ -11,27 +11,32 @@ case class VTabby(width: Int = 80, isAcross: Boolean = false, marginSize: Int =
 @RunWith(classOf[JUnit4])
 class TabulatorTest {
 
+  // à̧̄ is composed using 3 combining characters
+  val complexA = "a\u0300\u0304\u0327"
+  // 𝔟 is stored as 2 characters, but displayed as 1 grapheme
+  val frakturB = "\uD835\uDD1F"
+
   @Test def oneliner() = {
     val sut   = Tabby()
-    val items = List("a", "b", "c")
+    val items = List(complexA, frakturB, "c")
     val res   = sut tabulate items
     assert(res.size == 1)
     assert(res(0).size == 1)
-    assert(res(0)(0) startsWith "a")
+    assert(res(0)(0) startsWith complexA)
     assert(res(0)(0) endsWith "c")
   }
   @Test def twoliner() = {
     val sut   = Tabby(width = 40)
-    val items = List("a" * 15, "b" * 15, "c" * 15)
+    val items = List(complexA * 15, frakturB * 15, "c" * 15)
     val res   = sut tabulate items
     assert(res.size == 2)
```

Comment:
The fellow who wrote this was lazy and didn't use `assertEquals(2, res.size)`.

Reply:
Should I change all the asserts to use assertEquals?

Reply:
There are probably better good-third-issue ways to employ your time and talents.